src/FSharpVSPowerTools.Logic/VSLanguageService.fs

```diff
@@ -88,6 +102,10 @@ type VSLanguageService
             doc.Snapshot.TextBuffer = snapshot.TextBuffer
 
     static let initializationTime = DateTime.Now
+    let entityCache = EntityCache()
+
+    let solutionBuildEventListener = new SolutionBuildEventListener(serviceProvider)
+    do solutionBuildEventListener.ActiveConfigChanged.Add(fun _ -> entityCache.Clear())
```

Comment:
Why do we clear the cache here? All of the cache clearance is currently done at https://github.com/fsprojects/VisualFSharpPowerTools/blob/0a221963a73904e232a08565e5c6ed220ea13352/src/FSharpVSPowerTools.Logic/ProjectFactory.fs#L96-L99.